### PR TITLE
Add top level result panel

### PR DESCRIPTION
**Pull request by jessie-ross**
This PR adds top level panel including:
- Total participants
- AggregateRecommendation

Screen Shot 2021-03-09 at 12 59 54 pm

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
- Manual testing with mock data (locally or on Vercel)

### CODE REVIEW

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx [hunk 1]
```diff
@@ -129,6 +171,31 @@ export default function ActualExperimentResults({
...
+  // ### Top Level Stats
+
+  const primaryMetricLatestAnalysesByStrategy = _.mapValues(
+    primaryMetricAssignmentAnalysesData.analysesByStrategyDateAsc,
+    _.last.bind(null),
+  )
+
+  const totalParticipants = Object.values(primaryMetricLatestAnalysesByStrategy)
+    .map((primaryMetricLatestAnalysis) => {
+      // istanbul ignore next; trivial: If there is a missing strategy, count it as zero.
```
Comment (yanirs) [inline src/components/experiments/single-view/results/ActualExperimentResults.tsx:183]: I think it's worth testing.

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx [hunk 2]
```diff
@@ -129,6 +171,31 @@ export default function ActualExperimentResults({
...
   ]
 
+  // ### Top Level Stats
+
+  const primaryMetricLatestAnalysesByStrategy = _.mapValues(
+    primaryMetricAssignmentAnalysesData.analysesByStrategyDateAsc,
+    _.last.bind(null),
+  )
+
+  const totalParticipants = Object.values(primaryMetricLatestAnalysesByStrategy)
```
Comment (yanirs) [inline src/components/experiments/single-view/results/ActualExperimentResults.tsx:181]: Can you please explain why this is done this way? It's unclear why we need all this code to get the total participants.

Comment (jessie-ross) [reply src/components/experiments/single-view/results/ActualExperimentResults.tsx:182]: Ah, this should actually just be assignments, as it is the max of all participant totals.

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx [hunk 3]
```diff
@@ -130,6 +172,21 @@ export default function ActualExperimentResults({
...
 
+  // ### Top Level Stats
+
+  const primaryMetricLatestAnalysesByStrategy = _.mapValues(
+    primaryMetricAssignmentAnalysesData.analysesByStrategyDateAsc,
+    _.last.bind(null),
+  )
+  // We want the max participants so we use IttPure
+  const totalParticipants =
+    primaryMetricLatestAnalysesByStrategy[AnalysisStrategy.IttPure]?.participantStats['total'] ?? 0
```
Comment (yanirs) [inline src/components/experiments/single-view/results/ActualExperimentResults.tsx:183]: Why do we want the `IttPure` count? It seems likely to be confusing that the number doesn't match the latest variation sum, especially given that there's often a large difference between assigned and exposed numbers. 

If we change the wording below from `total participants` to `analyzed participants` and show the total from the `latestPrimaryMetricAnalysis` then everything should be consistent.

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx [hunk 4]
```diff
@@ -218,16 +274,54 @@ export default function ActualExperimentResults({
...
+                    />
+                  </Typography>
+                  <Typography variant='subtitle1'>
+                    <strong>primary metric</strong> recommendation
+                  </Typography>
+                </div>
+              </>
+            )}
+          </Paper>
+          {/* <Paper className={classes.summaryHealthPaper}></Paper> */}
```
Comment (withinboredom) [inline src/components/experiments/single-view/results/ActualExperimentResults.tsx:322]: Can we remove this?

**Review by withinboredom — APPROVED** [review]

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx (hunk 3 above)
Comment (jessie-ross) [reply src/components/experiments/single-view/results/ActualExperimentResults.tsx]: I've switched to analysed participants

File: src/components/experiments/single-view/results/ActualExperimentResults.tsx (hunk 4 above)
Comment (jessie-ross) [reply src/components/experiments/single-view/results/ActualExperimentResults.tsx:322]: Well it get's uncommented in the next PR in the stack

Comment (jessie-ross) [reply src/components/experiments/single-view/results/ActualExperimentResults.tsx:322]: Oh actually it doesn't, I'll just remove it for now

**Pull request merged**